### PR TITLE
fceux: fix for git tag change (upstream hot fix).

### PR DESCRIPTION
**Pull request by mjbudd77**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/fceux.rb [hunk 1]
```diff
@@ -3,7 +3,7 @@ class Fceux < Formula
   homepage "https://fceux.com/"
   url "https://github.com/TASEmulators/fceux.git",
       tag:      "fceux-2.6.0",
-      revision: "c075043727920bdd1250184c2b6bd33585c7ab43"
+      revision: "1ca8b8e9d0c401b8a9687b019af30aa088d1c4d2"
```
Comment (carlocab) [inline Formula/fceux.rb:6]: Why was this retagged? Do we need to publish new bottles or bump the revision?

Comment (mjbudd77) [reply Formula/fceux.rb:6]: The upstream repo retagged it due to something that was missed. So a hot fix was applied and re tagged. I was surprised that homebrew jumped on it so quickly

Comment (carlocab) [reply Formula/fceux.rb:6]: What was missed exactly? That determines whether we have to ship new builds and/or bump the revision (which will tell users who had the older `2.6.0` that their installation is outdated).

Comment (carlocab) [reply Formula/fceux.rb:6]: By the way, retagging your releases is generally not a nice thing to do to your downstream users. The [Git docs](https://git-scm.com/docs/git-tag#_on_re_tagging) have some rather strong words about retagging.

Comment (mjbudd77) [reply Formula/fceux.rb:6]: A small code patch. The application needs to be recompiled

Comment (mjbudd77) [reply Formula/fceux.rb:6]: The release wasn't even out for a full 24 hours and the patch was only a few lines. Next time I will stage it for a few days.

**Review by carlocab — APPROVED** [review]

File: Formula/fceux.rb (hunk 1 above)
Comment (carlocab) [reply Formula/fceux.rb:6]: It doesn't really matter how long a tag has been up: switching out your tags is likely to silently break someone's clone of your repository.

**Pull request closed without merging**